src/libraries/pretest.proj

```diff
@@ -78,7 +78,8 @@
              Projects="@(Project)"
              Properties="TargetFramework=$(BuildTargetFramework);%(Project.AdditionalProperties)"
              BuildInParallel="true"
-             ContinueOnError="ErrorAndStop" />
+             ContinueOnError="ErrorAndStop" 
+             Condition="'$(SkipBuildProjects)' != 'true'"/>
```

Comment:
What about instead directly invoking pretest.proj from CI and passing in the target that you care about?

Reply:
Yeah that makes sense. We could just call change the current call to /t: GenerateTestSharedFrameworkDepsFile.

Maybe on a follow-up PR so that he can merge?

Reply:
Yeah I'm fine with that.

Reply:
ok will do on next PR.